promql/functions.go

```diff
@@ -196,7 +196,7 @@ func instantValue(vals []parser.Value, out Vector, isRate bool) Vector {
 // The argument "s" is the set of computed smoothed values.
```

Comment:
Would you mind fixing this comment while you're here?

Reply:
ok , done.

Reply:
Thanks, none of these are sets anymore.

Reply:
ok，done